DESCRIPTION

```diff
@@ -11,7 +11,7 @@ Authors@R: c(
         person("Jordan", "Brooks", email = "jordan.brooks@insight-rx.com", role = "aut"),
         person("InsightRX", role = c("cph", "fnd")))
 Depends: R (>= 4.0.0)
-Imports: Rcpp (>= 0.12.9), BH, data.table, stringr, MASS,
+Imports: Rcpp (>= 1.0.13), BH, data.table, stringr, MASS,
```

Comment:
unrelated, saw that we hadn't updated this version requirement (we did in LinkingTo)